lib/fs.js

```diff
@@ -922,15 +922,15 @@ function stat(path, options = { bigint: false }, callback) {
   binding.stat(pathModule.toNamespacedPath(path), options.bigint, req);
 }
 
-function fstatSync(fd, options = {}) {
+function fstatSync(fd, options = { bigint: false }) {
   validateInt32(fd, 'fd', 0);
   const ctx = { fd };
   const stats = binding.fstat(fd, options.bigint, undefined, ctx);
```

Comment:
For @devsnek 's concerns, how about changing this to:
```suggestion
   const stats = binding.fstat(fd, options.bigint || false, undefined, ctx);
```

Reply:
in a future change we could do `{ bigint = false } = {}` for all of them